UI: Enforce Web3 private key are 0x prefixed

Seen when testing on staging with my own shop.  I'm not sure if this effects any other shoop:

    [ERROR] dshop.offerProcessor: Offer creation for shop 11 failed: { Error: invalid hexlify value (argument="value", value="[non-0x-prefixed-hex-string]", code=INVALID_ARGUMENT, version=bytes/5.0.3)
        at Logger.makeError (/app/node_modules/@ethersproject/logger/lib/index.js:179:21)
        at Logger.throwError (/app/node_modules/@ethersproject/logger/lib/index.js:188:20)
        at Logger.throwArgumentError (/app/node_modules/@ethersproject/logger/lib/index.js:191:21)
        at Object.hexlify (/app/node_modules/@ethersproject/bytes/lib/index.js:189:19)
        at new SigningKey (/app/node_modules/@ethersproject/signing-key/lib/index.js:19:65)
        at new Wallet (/app/node_modules/@ethersproject/wallet/lib/index.js:78:36)
        at Queue.processor (/app/backend/queues/makeOfferProcessor.js:99:20)
        at process._tickCallback (internal/process/next_tick.js:68:7)
      reason: 'invalid hexlify value',
      code: 'INVALID_ARGUMENT',
      argument: 'value',
      value:
       '[non-0x-prefixed-hex-string]' }

Appears to be because my privkey was not prefixed with `0x`

@franckc possibly related to our move to ethers.js?

--- a/backend/routes/shops.js
+++ b/backend/routes/shops.js
@@ -33,6 +33,7 @@ const formidable = require('formidable')
 const https = require('https')
 const http = require('http')
 const mv = require('mv')
+const { isHexPrefixed, addHexPrefix } = require('ethereumjs-util')
 
 const { configureShopDNS, deployShop } = require('../utils/deployShop')
 const { DSHOP_CACHE, IS_PROD } = require('../utils/const')
@@ -382,7 +383,9 @@ module.exports = function (router) {
       deliveryApi: req.body.printfulApi ? true : false
     }
     if (req.body.web3Pk && !config.web3Pk) {
-      config.web3Pk = req.body.web3Pk
+      config.web3Pk = isHexPrefixed(req.body.web3Pk)
+        ? req.body.web3Pk
+        : addHexPrefix(req.body.web3Pk)
     }
     const shopResponse = await createShop({
       networkId: network.networkId,

